### PR TITLE
2022.03.0 release

### PR DESCRIPTION
**Pull request by mcfadden8**

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,66 +1,48 @@
...
+
+## Fixes
+- Fixed a cmake install config issue so that now users can find a package of Umpire with a version constraint.
+- Fix `ResourceManager::isAllocator` to work for resources
+- Fix comparison operators for `TypedAllocators`
+- Fix host and device Allocator ID overlap
+- Remove null and zero-byte pool from list of valid allocators
+
+## New Configuration Options
+- The `UMPIRE_ENABLE_DEVICE_ALLOCATOR` option was added to control whether or not the DeviceAllocator class is included in the library.  The default is "On".
```
Comment (kab163) [inline RELEASE_NOTES.md]: I think I changed the default to be off

Comment (mcfadden8) [reply RELEASE_NOTES.md]: Oh really?  Oops

Comment (mcfadden8) [reply RELEASE_NOTES.md]: Fixed

Comment (kab163) [reply RELEASE_NOTES.md]: Thanks! Yea, I just confirmed it is off by default. I think it was originally On by default, but then when I added some of my DA updates, that caused some problems. (So I don't think it was well documented that it was subsequently turned Off by default!)

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,28 +6,32 @@ The format is based on [Keep a
...
-- Umpire now requires C++14.
+- C/FORTRAN API is now auto generated
+
+- Make umpire-config.cmake package relocatable
+
+- Add `UMPIRE_ENABLE_DEVICE_ALLOCATOR` option to disable Device Allocator which defaults
+  to "Off", to control whether or not the DeviceAllocator class is included in the
+  library.
+
+- Use blt namespace for hip targets (#692)
```
Comment (kab163) [inline CHANGELOG.md:34]: I am not a markdown expert by any means, but does a `#` symbol here cause markdown to get confused, or does it look ok?

Comment (mcfadden8) [reply CHANGELOG.md:34]: Good question.  It seems to be okay though (see: https://github.com/LLNL/Umpire/blob/release/2022.03.0/CHANGELOG.md)

**Comment by kab163** [comment]
Do you think we have to mention anything about the change in release versioning? Or will people just kinda understand? (Note about how it should correspond with same versions of RAJA/CHAI?)

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,28 +6,32 @@ The format is based on [Keep a
...
+## [v2022.03.0 - 2022-03-14]
 
-### Added
+### Build Changes and Updates
 
-- Added a getDeviceAllocator function that allows users to get a DeviceAllocator object
-  from the kernel without explicitly passing the allocator to the kernel first.
-
-- Added a reset function to the DeviceAllocator so that old data can be rewritten.
+- Umpire now requires minimum cmake version to be 3.14
```
Comment (davidbeckingsale) [inline CHANGELOG.md]:
```suggestion
- Umpire now requires minimum CMake version to be 3.14
```

File: RELEASE_NOTES.md [hunk 4]
```diff
@@ -1,3 +1,49 @@
+# v2022.03.0
+
+## Changes Impacting Builds
+This release of Umpire contains new build requirements including:
+
+- C++14 is now required to build Umpire
+- `cmake` version 3.14 or later is required
```
Comment (davidbeckingsale) [inline RELEASE_NOTES.md]:
```suggestion
- CMake version 3.14 or later is required
```

File: RELEASE_NOTES.md [hunk 5]
```diff
@@ -1,3 +1,49 @@
...
+
+## Changes Impacting C/Fortran
+- The `cmake` object library for `C/FORTRAN` interface has been reorganized.
```
Comment (davidbeckingsale) [inline RELEASE_NOTES.md]:
```suggestion
- The CMake object library for `C/FORTRAN` interface has been reorganized.
```

File: RELEASE_NOTES.md [hunk 6]
```diff
@@ -1,3 +1,49 @@
...
+  (**NOTE**: *This is a breaking change since the include paths are now different.*)
+
+## New Interfaces
+- Added a `getDeviceAllocator` function that allows users to get a `DeviceAllocator` object from the kernel without explicitly passing the allocator to the kernel first.
+- Added a `reset` function to the `DeviceAllocator` so that old data can be rewritten.
+- Expose `PREFETCH` operations registered with the `MemoryOperationRegistry` with a new `ResourceManager::prefetch` method.
+
+## Removed Interfaces
```
Comment (davidbeckingsale) [inline RELEASE_NOTES.md:21]: I think there should be a blank line after all these headings.

**Review by davidbeckingsale — CHANGES_REQUESTED** [review]

File: RELEASE_NOTES.md (hunk 6 above)
Comment (mcfadden8) [reply RELEASE_NOTES.md:21]: Done

**Review by davidbeckingsale — APPROVED** [review]

**Pull request merged**